### ISSUE TITLE
Cameras stopped working

### ISSUE THREAD

**Issue by cantchooseaname8** [dgreif/ring#349]
# Bug Report

### Describe the Bug
 Everything works on version 9.0.0. The camera live stream and snapshots all worked. Any version above that breaks the live stream and snapshots. Reverting back to version 9.0.0 fixes the issue. 

### To Reproduce
 - upgrade to any version above 9.0.0
 - snapshots will not update and live stream results in no response

### Screenshots/Logs
 I really wish I could be more helpful with logs. I have no clue how to access them. My homebridge knowledge is minimal. Perhaps if anyone else has this issue, they can report as well. 

### Homebridge Ring Config
Post homebridge-ring platform config  _without sensitive information_
```

HIGHLIGHT THIS TEXT AND POST CONFIG HERE
DO NOT INCLUDE YOUR REFRESH TOKEN OR EMAIL/PASSWORD (replace them with xxx)

```

### Environment

 - OS: Debian
 - Node.js: 12.16.3 
 - NPM: 6.14.5 
 - homebridge-ring: 9.1.0
 - homebridge: not sure. It was recently updated.

**Comment by JazzerAlto** [comment]
What command did you use to revert?  I am having issues as well but unable to roll back.

**Comment by cantchooseaname8** [comment]
> What command did you use to revert? I am having issues as well but unable to roll back.

npm i -g homebridge-ring@9.0.0

**Comment by JazzerAlto** [comment]
Not sure why but it didn't work for me.  I had to do a restore from backup.  It wouldn't roll back cleanly with that command.

**Comment by cantchooseaname8** [comment]
I also just noticed that the alarm hangs in versions after 9.0.0 and doesn’t arm.

**Comment by dgreif** [comment]
@cantchooseaname8  it sounds like your homebridge our HomeKit got into a weird state. If you can check your logs, you may find an error. I suggest you reboot homebridge, your home hub, and your phone. See if that gets you fixed up

**Comment by cantchooseaname8** [comment]
> @cantchooseaname8 it sounds like your homebridge our HomeKit got into a weird state. If you can check your logs, you may find an error. I suggest you reboot homebridge, your home hub, and your phone. See if that gets you fixed up

It’s working now. I’m really not sure what was going on. I restarted everything (phone, hub, Homebridge, pi). I then reinstalled the newest version and that still didn’t work. So I restarted everything again and it still didn’t work. The cameras wouldn’t load and showed no response. I just let it sit for a little while and then everything came back online. Weird situation. Maybe it had something to do with loading Ring’s servers or something but everything seems to be working great again! Thank you!!!
